recipes/acpia-tools/recipe.yaml

```diff
@@ -0,0 +1,60 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+context:
+  version: "2025_04_04"
+
+package:
+  name: acpica-tools
+  version: ${{ version | replace('_', '') }}
+
+source:
+  url: https://github.com/acpica/acpica/archive/refs/tags/R${{ version }}.tar.gz
+  sha256: 9991ec103b3660d17715780406ee7409f705cf87ac55e9a32374affe1a6f275a
+  patches:
+    - patches/buggy-acpicf.h.patch
+
+build:
+  number: 0
+  script:
+    - if: unix
+      then: |
+        make PREFIX=$PREFIX OPT_CFLAGS="-DACPI_PACKED_POINTERS_NOT_SUPPORTED"
+        make install PREFIX=$PREFIX
+      else: |
+        make PREFIX=%LIBRARY_PREFIX% OPT_CFLAGS="-DACPI_PACKED_POINTERS_NOT_SUPPORTED"
+        make install PREFIX=%LIBRARY_PREFIX%
+
+requirements:
+  build:
+    - if: unix
+      then:
+        - ${{ compiler('c') }}
+        - ${{ stdlib('c') }}
+        - bison
+        - flex
+        - m4
+      else:
+        - ${{ compiler('m2w64_c') }}
+        - ${{ stdlib('m2w64_c') }}
```

Comment:
Results in:
```
gcc_win-64 14.* cannot be installed because there are no viable options:
```

I wonder if pinning to the newer 15.1.0 would help.

https://prefix.dev/channels/conda-forge/packages/gcc_win-64

Reply:
Probably not, `gcc` isn't specified manually but as part of the MinGW compiler toolchain.

Reply:
Probably clutching at straws 😔

Reply:
No dice, tried locally with a `pixi.toml`.

Reply:
Looks like the solution is in https://github.com/conda-forge/m2-binary-packages-feedstock/issues/12#issuecomment-2664238921.